crescent/commands/options.py

```diff
@@ -39,14 +41,23 @@
     "ClassCommandOption",
 )
 
-OPTIONS_TYPE_MAP: dict[type[OptionTypesT], OptionType] = {
+
+class MemberInt(int):
```

Comment:
MemberID?

Reply:
I dont think that name fits here because there isn't an ID.

Reply:
Oh didn't realize that

Reply:
Oh I see. Maybe `MemberType` instead? `MemberInt` doesn't feel right.

Reply:
I think im gonna do `MemberOptionType`

Reply:
done